core/core-impl/src/main/java/ai/timefold/solver/core/impl/solver/DefaultSolver.java

```diff
@@ -234,16 +226,34 @@ public void solvingStarted(SolverScope<Solution_> solverScope) {
         solverScope.startingNow();
         solverScope.getScoreDirector().resetCalculationCount();
         super.solvingStarted(solverScope);
-        int startingSolverCount = solverScope.getStartingSolverCount() + 1;
+        var startingSolverCount = solverScope.getStartingSolverCount() + 1;
         solverScope.setStartingSolverCount(startingSolverCount);
-        logger.info("Solving {}: time spent ({}), best score ({}), environment mode ({}), "
-                + "move thread count ({}), random ({}).",
+        registerSolverSpecificMetrics();
+        logger.info("Solving {}: time spent ({}), best score ({}), "
+                + "environment mode ({}), move thread count ({}), random ({}).",
                 (startingSolverCount == 1 ? "started" : "restarted"),
                 solverScope.calculateTimeMillisSpentUpToNow(),
                 solverScope.getBestScore(),
                 environmentMode.name(),
                 moveThreadCountDescription,
                 (randomFactory != null ? randomFactory : "not fixed"));
+        logger.info(
```

Comment:
I keep thinking... isn't it enough to print this just once, either at start or finish?
It will end up in the log, which means it's just duplication if the same information is there twice.